admin/js/common.js

```diff
@@ -562,13 +565,13 @@ var qTranslateX=function(pg)
 								}
 							}
 						}else{
-							continue;
+							continue;
```

Comment:
I found this: https://help.github.com/articles/dealing-with-line-endings/#platform-all
My system wide Git configuration was set to "autocrlf = input" which is why it converted the linebreaks.